modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/SourceDataRenderer.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import classNames from 'classnames';
```

Comment:
@pedro-oliveira446 You can use it to replace this [one](https://github.com/liferay/liferay-portal/blob/0d251126abe828f08269a278d2b81b2b83111a1d/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Fields.tsx#L105-L118) too. It was mentioned by Camacho before.

Reply:
@naraandrade,
[here](https://github.com/liferay/liferay-portal/blob/0d251126abe828f08269a278d2b81b2b83111a1d/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Fields.tsx#L105-L118) the entire object is received as a parameter, in my case only the boolean field. This is because the picklist FDS table is not yet in react so the rendering is done differently. That's why I can't reuse it there. The next story is the system object validations, since the FDS table is also in react, I'm going to create a generic function and use it there too.

Reply:
I did it in this [commit](https://github.com/liferay-objects/liferay-portal/pull/2558/commits/920d0f985209d106ab343dcf31bb821d3c9e366c) .